chat/chat.py

```diff
@@ -94,6 +94,12 @@ class ChatXBlock(StudioEditableXBlockMixin, XBlock):
         scope=Scope.content,
     )
 
+    enable_restart_button = Boolean(
+        display_name=_("Enable restart button"),
+        default=False,
```

Comment:
Nit: Is there a reason why a dropdown is used instead of a checkbox? True/False feel a bit unnatural. If the dropdown has to be used, why not Yes/No?